src/jit/codegenxarch.cpp

```diff
@@ -2651,9 +2638,14 @@ CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
         getStructReturnRegisters(type0, type1, &retReg0, &retReg1);
 
         assert(retReg0 != REG_NA && retReg1 != REG_NA);
-
+        
+        // Copy of a struct to a well typed location on the stack. No need to call the 
+        // GC-ness of each slot. Need to make the store of the two slots
+        // atomic for the purposes of GC.
+        getEmitter()->emitDisableGC();
```

Comment:
It's not clear WHY you need to disable GC. Where is the GC hole that occurs without disabling it? The call presumably makes both returned registers go live.